examples/api/lib/painting/borders/border_side.0.dart

```diff
@@ -0,0 +1,204 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
This example needs a test.  See some of the other tests in examples/api/test for guidance. They're pretty simple tests: you could just verify that the expected `TestBox`es are in the widget tree after you pump a `BorderSideExample`.